etc/schema.json

```diff
@@ -3729,13 +3723,64 @@
             },
             "additionalProperties": false
         },
+        "pop3": {
+            "type": "object",
+            "optional": true,
+            "properties": {
+                "command": {
+                    "type": "object",
+                    "optional": true,
+                    "properties": {
+                        "keyword": {
+                            "type": "string"
```

Comment:
could you add a description for these fields as well ?

Reply:
What kind of description do you want? They are the pop3 command keyword and args.

Reply:
something like `a pop3 command, for example USER or STAT` ?